jobs/haproxy/spec

```diff
@@ -607,6 +609,12 @@ properties:
   ha_proxy.backend_http_health_port:
     description: Port to check the backend health on
     default: 8080
+  ha_proxy.backend_health_fall:
```

Comment:
It's probably worth mentioning that this is ignore unless backend_use_http_health is true (and same for `backend_health_rise`)

Reply:
Update the description to include the note about the property being ignored when haproxy.backend_use_http_health is false.